
# PRINCY FRANCIS
[üìß JHU](mailto:pfranc10@jh.edu) ‚Ä¢ [üìß NIH](mailto:francisp2@nih.gov)  ‚Ä¢ [üìß Gmail](mailto:princyfrancis24@gmail.com) <br>
‚òè +1 301 646 4414 ‚Ä¢ MD, USA
<br><br>

## EXPERIENCE
‚Ä¢ **Special Volunteer in Bioinformatics** [_2019 - present_] <br>
Stem Cell Translation Laboratory, NCATS, NIH, Rockville, MD, USA

‚Ä¢ **Senior Research Fellow** [_2013 - 2015_] <br>
Mayo Clinic Cancer Center, Scottsdale, AZ, USA <br>

‚Ä¢ **Post-Doctoral Researcher** [_2007 - 2012_] <br>
Genetics Branch, NCI, NIH, Bethesda, MD, USA <br>
<br>

## EXPERTISE
### BIOINFORMATICS
‚Ä¢ **NGS data analysis:** Sequence assembly, sequence alignment, genome annotation, variant calling, WGS, WES, bulk RNA-seq, single-cell RNA-seq, miRNA-seq, ChIP-seq, methyl-seq
<br>‚Ä¢ **Bioinformatics tools:** FASTQC, Trimmomatic, SRA Toolkit, BWA, Bowtie, Tophat, Cufflinks, SAMtools, GATK, Picard, PLINK, Freebayes, VCFtools, VarScan, ANNOVAR, VEP, snpEff, Delly, SVprops, IGV, MACS, PeakSeq, SPAdes, VCAKE, Edena, Velvet, Prokka, RSEM, Galaxy Tools, genome browsers (UCSC, Ensembl, IGV), R/Bioconductor packages (DESeq, Seurat), and various tools for analysis of miRNAs (miRDeep, miRDB, miRTAR, sRNAtoolbox), microarray data (Agilent CGH Analytics/Genomic Workbench, BioDiscovery Nexus Expression/Copy Number, BeadStudio), protein structure (I-TASSER, Phyre2, ConSurf), telomere length (Telseq, Telomerecat), biological pathways and functions (CytoScape, BioGrid, DAVID EASE), molecular phylogeny (MEGAX, MUSCLE, ClustalW, T-Coffee, MAFFT)
<br>‚Ä¢ **Programming languages:** Python, Java, CGI, R, Shell
<br>‚Ä¢ **Web design:** PHP, Perl DBI, HTML5/CSS3, Javascript/JQuery
<br>‚Ä¢ **Database management systems:** MySQL, GMOD/Chado
<br>‚Ä¢ **Visualisation:** R, Jupyter
<br>‚Ä¢ **Version control:** Git and GitHub
<br>‚Ä¢ **Operating systems:** Unix/Linux systems including the NIH HPC Biowulf cluster environments

### GENOMICS | MOLECULAR BIOLOGY | CELL BIOLOGY
‚Ä¢ **Sequencing technologies and genomic assays:** Illumina, Ion Torrent, library preparation (exome-seq, RNA-seq, miRNA-seq), Illumina BeadArrays, allele-specific SNP-arrays, gene/miRNA expression microarrays, arrayCGH
<br>‚Ä¢ **Cell and tumor biology:** In vitro osteoblast differentiation assays, stable/transient transfections (Lipofectamine-2000, siPORT, RNAiMax), alkaline phosphatase activity, Matrigel invasion assays, wound healing assays, cell proliferation assays
<br>‚Ä¢ **miRNA biology:** Enrichment from tissues/cell, TaqMan assays for primary/mature miRNA, interference technologies (mimics, hairpin-inhibitors, antisense-oligonucleotides), in vitro manipulation, target identification (loss-of-function/gain-of-function experiments, biotin-tagged mimics), target validation (luciferase reporter assays), miRNA lentiviral expression and inhibitor vectors
<br><br>


## EDUCATION
‚Ä¢ **MS Bioinformatics** [_2018 - 2021_] <br>
Advanced Academic Programs, Krieger School of Arts & Sciences, Johns Hopkins University, Baltimore, MD, USA

‚Ä¢ **PhD Experimental Oncology** [_2003 - 2007_] <br>
Department of Oncology, Institute for Clinical Sciences, Lund University Hospital, Lund, Sweden

‚Ä¢ **MSc Applied Genetics** [_1999 - 2001_] <br>
Centre for Applied Genetics, Bangalore University, Jnana Bharathi, Bangalore, India

‚Ä¢ **BSc Chemistry, Zoology, Microbiology** [_1996 - 1999_] <br>
B.H.S. First Grade College, Bangalore University, Jayanagar, Bangalore, India
<br><br>

## ADDITIONAL TRAINING
‚Ä¢ **Lab Assistant** [_2001 ‚Äì 2003_] <br>
Department of Biochemistry, Indian Institute of Science (IISc), Bangalore, India <br>
  
‚Ä¢ **Computational Analysis of Microarray Data** [_2001_] <br>
Department of Cell Biology and Microbiology, IISc, Bangalore, India <br>
  
‚Ä¢ **Advanced Certification Course in Bioinformatics** [_2001_] <br>
Manvish Infotech Limited, Bangalore, Karnataka, India
<br><br>

## PUBLICATIONS
‚Ä¢ Gindin Y, Jiang Y, Francis P, Walker RL, Abaan OD, Zhu YJ, Meltzer PS. _Mir-23a impairs bone differentiation in osteosarcoma via down-regulation of GJA1._ Front Genet. 2015 Jul 2.

‚Ä¢ Jones MF, Hara T, Francis P, Li XL, Bilke S, Zhu Y, Pineda M, Subramanian M, Bodmer WF, Lal A. _The CDX1-microRNA-215 axis regulates colorectal cancer stem cell differentiation._ Proc Natl Acad Sci U S A. 2015 Mar 3.

‚Ä¢ Subramanian M, Francis P, Bilke S, Li XL, Hara T, Lu X, Jones MF, Walker RL, Zhu Y, Pineda M, Lee C, Varanasi L, Yang Y, Martinez LA, Luo J, Ambs S, Sharma S, Wakefield LM, Meltzer PS, Lal A. _A mutant p53/let-7i-axis-regulated gene network drives cell migration, invasion and metastasis._ Oncogene. 2014 Mar 24.

‚Ä¢ Egan JB, Barrett MT, Champion MD, Middha S, Lenkiewicz E, Evers L, Francis P, Schmidt J, Shi CX, Van Wier S, Badar S, Ahmann G, Kortuem KM, Boczek NJ, Fonseca R, Craig DW, Carpten JD, Borad MJ, Stewart AK. _Whole Genome Analyses of a Well-Differentiated Liposarcoma Reveals Novel SYT1 and DDR2 Rearrangements._ PLoS One. 2014 Feb 5.

‚Ä¢ Li XL, Hara T, Choi Y, Subramanian M, Francis P, Bilke S, Walker RL, Pineda M, Zhu Y, Yang Y, Luo J, Wakefield LM, Brabletz T, Park BH, Sharma S, Chowdhury D, Meltzer PS, Lal A. _A p21-ZEB1 Complex Inhibits Epithelial-Mesenchymal Transition through the MicroRNA 183-96-182 Cluster._ Mol Cell Biol. 2014 Feb.

‚Ä¢ Beachy SH, Onozawa M, Chung YJ, Slape C, Bilke S, Francis P, Pineda M, Walker RL, Meltzer P, Aplan PD. _Enforced expression of Lin28b leads to impaired T-cell development, release of inflammatory cytokines, and peripheral T-cell lymphoma._ Blood. 2012 Aug 2.
			
‚Ä¢ Skubitz KM, Francis P, Skubitz AP, Luo X, Nilbert M. _Gene expression identifies heterogeneity of metastatic propensity in high-grade soft tissue sarcomas._ Cancer. 2012 Jan 17.
			
‚Ä¢ Isinger-Ekstrand A, Johansson J, Ohlsson M, Francis P, Staaf J, J√∂nsson M, Borg A, Nilbert M. _Genetic profiles of gastroesophageal cancer: combined analysis using expression array and tiling array--comparative genomic hybridization._ Cancer Genet Cytogenet. 2010 Jul 15.
			
‚Ä¢ Wallin A, Francis P, Nilbert M, Svanvik J, Sun XF. _Gene expression profile of colon cancer cell lines treated with SN-38._ Chemotherapy. 2010 Feb 24.
			
‚Ä¢ Carneiro A, Francis P, Bendahl PO, Fernebro J, Akerman M, Fletcher C, Rydholm A, Borg A, Nilbert M. _Indistinguishable genomic profiles and shared prognostic markers in undifferentiated pleomorphic sarcoma and leiomyosarcoma: different sides of a single coin?_ Lab Invest. 2009 Mar 16.
			
‚Ä¢ Francis P, Namlos HM, Muller C, Eden P, Fernebro J, Berner JM, Bjerkehagen B, Akerman M, Bendahl PO, Isinger A, Rydholm A, Myklebost O, Nilbert M. _Diagnostic and prognostic gene expression signatures in 177 soft tissue sarcomas: hypoxia-induced transcription profile signifies metastatic potential._ BMC Genomics. 2007 Mar 14.
			
‚Ä¢ Fernebro J, Francis P, Eden P, Borg A, Panagopoulos I, Mertens F, Vallon- Christersson J, Akerman M, Rydholm A, Bauer HC, Mandahl N, Nilbert M. _Gene expression profiles relate to SS18/SSX fusion type in synovial sarcoma._ Int J Cancer. 2006 Mar 1.

‚Ä¢ Nilbert M, Meza-Zepeda LA, Francis P, Berner JM, Namlos HM, Fernebro J, Myklebost O. _Lessons from genetic profiling in soft tissue sarcomas._ Acta Orthop Scand Suppl. 2004 Apr
			
‚Ä¢ Francis P, Fernebro J, Eden P, Laurell A, Rydholm A, Domanski HA, Breslin T, Hegardt C, Borg A, Nilbert M. _Intratumor versus intertumor heterogeneity in gene expression profiles of soft-tissue sarcomas._ Genes Chromosomes Cancer. 2005 Jul.
			
‚Ä¢ Pillai B, Verma J, Abraham A, Francis P, Kumar Y, Tatu U, Brahmachari SK, Sadhale PP. _Whole genome expression profiles of yeast RNA polymerase II core subunit, Rpb4, in stress and nonstress conditions._ J Biol Chem. 2003 Jan 31.
